webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -28,6 +28,13 @@ export const bindMethods = (context, methods) => {
   });
 };
 
+/**
+ * Removes slashes from the beggining and end of the path
+ * @param {String} path - the path that should be removed of slashes
+ */
+export const removeLastSlashFromPath = path =>
+  path ? path.replace(/\/+$/g, '') : path;
```

Comment:
`replace` performance is very poor, comparing with slicing: 

![replace-vs-slicing](https://user-images.githubusercontent.com/11807069/51255187-640a2800-19ab-11e9-986f-fc0d729b5dca.png)

Reply:
Changed.